WireAuthentication/Sources/WireAuthenticationUI/Views/Login/LoginViaEmail/LoginViaEmailView.swift

```diff
@@ -26,10 +28,9 @@ public protocol LoginViaEmailBuilder {
 }
 
 public struct LoginViaEmailView: View {
-
     @ObservedObject var viewModel: LoginViaEmailViewModel
 
-    @State private var password = ""
+    @State var password: String = ""
```

Comment:
suggestion: move this to the view model. If you keep in here, it should be marked as `private` (SwiftUI best practice)

Reply:
made it `private`